package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "@babel/runtime": "7.8.3",
     "bluebird": "2.9.25",
-    "forest-express": "5.6.1",
+    "forest-express": "7.0.1",
```

Comment:
It would be better with 7.0.2, isn't it?
The last on npm: https://www.npmjs.com/package/forest-express

Reply:
Good catch. This PR is a kind of a rollback PR, so I don't want to introduce changes. When the original PR was made it was on 7.0.1. I guess we could release this later (maybe in a small PR later that will not trigger a new version).